src/components/__tests__/TodoList.test.tsx

```diff
@@ -0,0 +1,34 @@
+import { fireEvent, render, screen } from '@testing-library/react';
+import TodoList from '../TodoList';
+
+describe('TodoList', () => {
+  it('should render without errors', () => {
+    render(<TodoList />);
+
+    expect(screen.getByTestId('container')).toBeInTheDocument();
```

Comment:
(Remove Test) No need for this test. This 'functionality' it tested in all other tests if the container doesn't render then you can't test anything at all